jsx-runtime/src/encode.js

```diff
@@ -0,0 +1,36 @@
+const ENCODED_ENTITIES = /["&<]/;
+
+/** @param {string} str */
+export function encodeEntities(str) {
```

Comment:
This function is just copied over from `preact-render-to-string` https://github.com/preactjs/preact-render-to-string/blob/17f839c47309d7de35cc407c25b016a0dcc325c8/src/util.js#L11